### ISSUE TITLE
Minimap in FTB Chunks appears to be broken again

### ISSUE THREAD

**Issue by nate34k** [Pannoniae/sodium-forge#60]
### Expected Behavior
Minimap loads as normal in 2.0 fix

### Actual Behavior
Minimap does not load the environment data

### Reproduction Steps
Launch game with sodium-forge 2.1.x and latest version of FTB Chunks
 any relevant screenshots or videos to this issue as you feel necessary to explain the issue.

If the fix for this is not relevant to this mod, please close this Issue and I will forward it to the other mod team, but since I saw it was fixed on this end before I figured that it may be a similar issue

**Comment by Pannoniae** [comment]
Yeah, it's been removed because the other guy didn't want to setup deps. I'll fix :)
